packages/web-components/src/components/va-back-to-top/va-back-to-top.css

```diff
@@ -1,15 +1,16 @@
 @import '../../../../../node_modules/animate.css/source/sliding_entrances/slideInUp.css';
 @import '../../../../../node_modules/animate.css/source/sliding_exits/slideOutDown.css';
 @import '../../mixins/buttons.css';
-:host {
+
+#marginWrapper {
```

Comment:
Using this extra wrapping element gives us the isolation we need to ensure the margin doesn't get removed by the higher specificity CSS from the global styles.

[From some documentation on styling + Shadow DOM](https://developers.google.com/web/fundamentals/web-components/shadowdom):

> One gotcha with `:host` is that rules in the parent page have higher specificity than `:host` rules defined in the element. That is, outside styles win.